packages/backend/src/modules/dashboard/Board.ts

```diff
@@ -0,0 +1,36 @@
+import mongoose from 'mongoose'
+
+const { ObjectId } = mongoose.Types
+
+const boardSchema = new mongoose.Schema({
+  _id: ObjectId,
+  title: String,
+  description: String,
+  createdAt: Date,
+  status: String,
+  users: [
+    {
+      _id: ObjectId,
+      username: String,
+      avatarId: Number
+    }
+  ],
+  columns: [
```

Comment:
create schema for Column, use embeded document
see:
https://www.youtube.com/watch?v=b59Pdt5LJG8&ab_channel=WornOffKeys